core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -366,6 +366,24 @@ public static long getBytes(PropertyKey key) {
   }
 
   /**
+   * Gets the time in the given key.
+   *
+   * @param targetUnit the target unit set by user
+   * @param key the key to get the value for
+   * @return the time transformed from milliseconds to the target unit
+   */
+  public static double getTime(String targetUnit, PropertyKey key) {
```

Comment:
can you add unit tests of this method into "ConfigurationTest.java"? You can follow how it is structured currently.